### DISCUSSION TITLE
Creation of isDirty() helper method

### DISCUSSION THREAD

**Discussion by sergebenard** [inertiajs/inertia#401] [Ideas]
#376 

It'd be a good thing to have a helper function `isDirty()` to know if any of the form's elements are dirty, in order to know if there needs to be a call to 'window.onBeforeUnload' to confirm the user's intent of leaving a page which is holding unsaved form data.

**Comment by reinink** [comment]
I've started a new PR for this feature: https://github.com/inertiajs/inertia/pull/660
